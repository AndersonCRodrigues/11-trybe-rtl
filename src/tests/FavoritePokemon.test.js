import { screen } from '@testing-library/react';
import { FavoritePokemon } from '../pages';
import renderWithRouter from '../renderWithRouter';

describe('<FavoritePokemon />', () => {
  it('testa  Página Favorites ', () => {
    renderWithRouter(<FavoritePokemon />);

    const heading = screen.getByRole('heading', { name: 'Favorite Pokémon', level: 2 });
    const paragrafo = screen.getByText('No favorite Pokémon found');

    expect(heading).toBeInTheDocument();
    expect(paragrafo).toBeInTheDocument();
  });

  it('mesma snapshot', () => {
    const { container } = renderWithRouter(<FavoritePokemon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
