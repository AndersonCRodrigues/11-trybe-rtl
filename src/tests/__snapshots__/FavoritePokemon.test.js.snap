// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritePokemon /> mesma snapshot 1`] = `
<div>
  <h2>
    Favorite Pokémon
  </h2>
  <div>
    <p>
      No favorite Pokémon found
    </p>
  </div>
</div>
`;
